.travis.yml

```diff
@@ -16,19 +16,18 @@ before_install:
   - cp config/secrets.travis.yml config/secrets.yml
```

Comment:
Changes look good. Just one small nitpick here..

Could we combine the secrets.travis.yml with the secrets.example.yml into one secrets.example.yml? (aka put in dummy secrets keys)

Then you dont have any specific travis configs in your app